application/models/Airports.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * Dummy model class that represents the airports our airline services 
+ */
+class Airports extends CI_Model
+{
+
+    // The mock airport data
+    var $data = array(
+        'L_YPR' => array(
+            'code' => 'YPR',
+            'location' => 'Prince Rupert Airport'),
+        'L_ZMT' => array(
+            'code' => 'ZMT',
+            'location' => 'Masset Airport'),
+        'L_YZP' => array(
+            'code' => 'YZP',
+            'location' => 'Sandspit Airport'),
+        'L_YZT' => array(
+            'code' => 'YZT',
```

Comment:
I think the code for this airport is `YXT` not `YZT`.
I'm looking at http://wacky.jlparry.com/info/airlines/kite

Reply:
so it is.  one day i'll learn to read.